sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -200,214 +182,234 @@ export class EventHubReceiver extends LinkEntity {
     consumerGroup: string,
     partitionId: string,
     eventPosition: EventPosition,
-    options?: EventHubConsumerOptions
+    options: EventHubConsumerOptions = {}
```

Comment:
The jsdocs on the constructor needs an update

Reply:
I think the JSDocs are still accurate, unless you mean more description around what `EventHubConsumerOptions` are. Even though we have a default option, to the user of this API it is still an optional parameter, and the type definition doesn't change.

Reply:
- The first param should be context not client
- The description "Instantiate a new receiver from the AMQP `Receiver`. Used by `EventHubClient`" is.... well.... wrong

Suggestion:
> Instantiates a receiver that can be used to receive events over a AMQP receiver link in either batching or streaming mode

Reply:
I pushed a fix